entrypoint

```diff
@@ -58,17 +62,24 @@ def init():
     with WEBDIR_CONF_PATH.open("w") as f:
         print(f"Copying webdir conf to {WEBDIR_CONF_PATH}...")
         f.write(WEBDIR_CONF)
+    print("Done.")
+
 
+def start(args):
+    """Run a local mercurial web server instance.
 
-def start():
-    """Run a local HG web server instance.
+    Args:
+        args (argparse.Namespace): Additional arguments passed to `start` from the
+            command line: hg_serve_port.
     """
-    subprocess.call(
+    hg_serve_port = args.hg_serve_port
+
+    subprocess.run(
         [
             "hg",
             "serve",
```

Comment:
Nit: if you wanted this to more closely match production hgssh, you would run `hg -R <repo> serve --stdio`. I'm not sure if that's necessary for this though.

Reply:
As per offline conversation, leaving this as is for now.